### PR TITLE
[to #48] remove mount metircs

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zeminzhou_@outlook.com>

### What problem does this PR solve?

Issue Number: to #48 

Problem Description: 
The mount has been removed, related codes should be removed.

### What is changed and how does it work?
Remove metrics code for mount.

### Code changes

- Has exported function/method change
- Has exported variable/fields change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test

### CODE REVIEW

File: cdc/cdc/metrics.go [hunk 1]
```diff
@@ -37,20 +36,12 @@ func init() {
...
 	initServerMetrics(registry)
-	// actor.InitMetrics(registry)
 	orchestrator.InitMetrics(registry)
 	p2p.InitMetrics(registry)
-	// Sorter metrics
-	// sorter.InitMetrics(registry)
-	// memory.InitMetrics(registry)
-	// unified.InitMetrics(registry)
-	// leveldb.InitMetrics(registry)
-	// redowriter.InitMetrics(registry)
```
Comment (zz-jason) [inline cdc/cdc/metrics.go:54]: how about just removing them instead of commenting them out?

Comment (zeminzhou) [reply cdc/cdc/metrics.go:54]: yes, we should remove them.

Comment (zeminzhou) [reply cdc/cdc/metrics.go:54]: yes, we should remove them.

**Review by pingyu — APPROVED** [review]
LGTM~

**Comment by pingyu** [comment]
`TestRegionWorkerExitWhenIsIdle` seems to be not stable in gh. The same test in Jenkins is successful.
I think we can leave it alone now and resolve later.

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**